### PR TITLE
Add return type of scoped_session.__call__

### LINKED ISSUE #136: scoped_session __call__ returns Any
**Describe the bug**
Calling scoped_session returns Any which is should not be the case

**Expected behavior**
A concrete type should be returned

**To Reproduce**
```py
session_factory = sessionmaker(bind=create_engine("foo", future=True))
reveal_type(scoped_session(session_factory)())
```

**Error**
```
Revealed type is "Any"
```

**Versions.**
 - OS: Linux
 - Python: 3.7.1
 - SQLAlchemy: 1.4.22
 - mypy: 0.910
 - SQLAlchemy2-stubs: 0.0.2a6

### PR DESCRIPTION
**Pull request by kasium**
Add return type of scoped_session.\_\_call__.
This resolves #136.

### Checklist

This pull request is:

- [ ] A documentation / typographical error fix
	- Good to go, no issue or tests are needed
- [X] A short code fix
	- please include the issue number, and create an issue if none exists, which
	  must include a complete example of the issue.  one line code fixes without an
	  issue and demonstration will not be accepted.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.   one line code fixes without tests will not be accepted.
- [ ] A new feature implementation
	- please include the issue number, and create an issue if none exists, which must
	  include a complete example of how the feature would look.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.

**Have a nice day!**

### CODE REVIEW

File: sqlalchemy-stubs/orm/scoping.pyi [hunk 1]
```diff
@@ -1,13 +1,15 @@
 from typing import Any
 from typing import Optional
 
+from ..orm import Session as Session
```
Comment (CaselIT) [inline sqlalchemy-stubs/orm/scoping.pyi:4]: I think it's best to avoid `as Session` since it signals mypy that it's a public export

Comment (kasium) [reply sqlalchemy-stubs/orm/scoping.pyi]: right, fixed

**Review by CaselIT — APPROVED** [review]

**Comment by CaselIT** [comment]
Thanks. will release shortly

**Comment by CaselIT** [comment]
Released as `v0.0.2a7`

**Pull request merged**